src/script/scripting_server.cpp

```diff
@@ -62,6 +62,9 @@ ServerScripting::ServerScripting(Server* server):
 
 	if (g_settings->getBool("secure.enable_security")) {
 		initializeSecurity();
+	} else {
+		warningstream << "Disabling mod security is deprecated, and should not be done. "
+				<< "Mods should use minetest.request_insecure_environment() instead" << std::endl;
```

Comment:
```suggestion
		warningstream << "Disabling mod security is deprecated and "
			<< "will be removed in a future release. Mods should use "
			<< "minetest.request_insecure_environment() instead" << std::endl;
```

Reply:
sfan5 is against removing this setting

How about:

> \!/ Disabling mod security should not be done, as it allows any mod to access the host computer. Mods should use minetest.request_insecure_environment() instead \!/

Reply:
that's okay as well.